go.mod

```diff
@@ -206,6 +206,7 @@ require (
 	github.com/gorilla/mux v1.8.0 // indirect
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/hashicorp/go-immutable-radix v1.3.1 // indirect
+	github.com/hashicorp/go-netaddrs v0.1.0 // indirect
```

Comment:
This should be direct now, so let's move this up to the section above and remove the `// indirect`. Running `go mod tidy` should do it.